.github/workflows/tests.yml

```diff
@@ -30,14 +30,13 @@ jobs:
     - name: setup environment
       run: |
         export num_proc=$(nproc)
-        echo "::set-env name=GTRUN_NO_SLURM::true"
```

Comment:
please change line 10 above to something that fits better, e.g. just `tests` instead of `perftests`